ticket_facts.view.lkml

```diff
@@ -0,0 +1,277 @@
+      type: number
+      sql: ${TABLE}.number_outbound_calls ;;
+    }
+
+    dimension: has_outbound_calls {
+      description: "Yes, if ticket has associated outbound calls"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${number_outbound_calls} > 0 ;;
+    }
+
+    dimension: number_inbound_calls {
+      description: "Number of inbound calls associated with ticket"
+      group_label: "Activity Counts"
+      type: number
+      sql: ${TABLE}.number_inbound_calls ;;
+    }
+
+    dimension: has_inbound_calls {
+      description: "Yes, if ticket has associated inbound calls"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${number_inbound_calls} > 0 ;;
+    }
+
+    dimension: number_inbound_emails {
+      description: "Number of inbound emails associated with ticket"
+      group_label: "Activity Counts"
+      type: number
+      sql: ${TABLE}.number_inbound_emails ;;
+    }
+
+    dimension: has_inbound_emails {
+      description: "Yes, if ticket has associated inbound emails"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${number_inbound_emails} > 0 ;;
+    }
+
+    dimension: involves_customer {
+      description: "Yes, if ticket has inbound or outbound activity"
+      group_label: "Ticket Details"
+      type: yesno
+      sql: ${number_inbound_calls} > 0 OR ${number_inbound_emails} > 0 OR ${number_outbound_calls} > 0 OR ${number_outbound_emails} > 0 ;;
+    }
+
+    dimension: is_first_touch_resolution_email {
+      description: "Yes, if inbound email was solved at first agent touch"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${ticket_source} LIKE 'Inbound Email%'
+         AND ${status} IN ('solved', 'closed')
+         AND ${is_ticket_merged} = false
+         AND (${number_outbound_emails} + ${number_outbound_calls}) = 1
+         AND ${is_parent_to_merged_tickets} = false ;;
+    }
+
+    dimension: is_eligible_for_first_touch_resolution_email {
+      description: "Yes, if inbound email was solved and has outbound activity"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${ticket_source} LIKE 'Inbound Email%'
+          AND ${status} IN ('solved', 'closed')
+          AND ${is_ticket_merged} = false
+          AND (${number_outbound_emails} + ${number_outbound_calls}) >= 1 ;;
+    }
+
+    dimension: is_first_touch_resolution_phone_call {
+      description: "Yes, if inbound phone call was solved at first agent touch"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${ticket_source} IN ('Inbound Call','Inbound Voicemail')
+           AND ${status} IN ('solved', 'closed')
+           AND ${is_ticket_merged} = false
+           AND ${number_inbound_calls} = 1
+           AND (${number_outbound_emails} + ${number_outbound_calls}) = 0
+           AND ${is_parent_to_merged_tickets} = false ;;
+    }
+
+    dimension: is_eligible_for_first_touch_resolution_phone_call {
```

Comment:
Don't really understand this business metric?

Reply:
same comment as above.